### DISCUSSION TITLE
Possibilità di assegnare compiti senza entrare ogni volta in agenda

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#213] [Idee e proposte]
Salve, sarebbe possibile inserire la possibilità di firmare e inserire argomenti e compiti direttamente dalla firma con data

**Comment by scuolagioco** [comment]
> Salve, sarebbe possibile inserire la possibilità di firmare e inserire argomenti e compiti direttamente dalla firma con data

E anche scegliere la data di consegna
